protos/dcs/mission/v0/mission.proto

```diff
@@ -343,6 +386,8 @@ message StreamEventsResponse {
     // @exclude 38 reserved for S_EVENT_TRIGGER_ZONE
     LandingQualityMarkEvent landing_quality_mark = 39;
     // @exclude 40 reserved for S_EVENT_BDA
+    ConnectEvent connect = 41;
```

Comment:
Wdyt, should we continue with the next number here, or leave some field numbers free for actual future DCS events?

Reply:
I think it is safer for us to chose a large random number. like 29583 or something that is extremely unlikely to be used by DCS itself.

Reply:
Do you know if this increases the size of the protobuf message?

Reply:
It doesn't.

Reply:
Done in a2993aeb2e3cd6a7f57f5d5f015b2f468e940509. I didn't go thaaatt high, my mind felt better if I could still spot that the number is a multiple of 1024